README.md

```diff
@@ -75,9 +81,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1.In its own stylesheet file
+2.In the head area of the index file within <style> </style>
+3.In the html code itself such as <p> <strong> this is a paragraph. </strong> </p>
 ```
```

Comment:
Using tags like `<strong>` isn't considered to be CSS. I think you were going for inline styling, like this...

```css
<div style="color:blue;">Hello</div>
```

As far as specificity goes, the order from most to least is (1) inline (2) <style></style> and (3) external stylesheet. While an external stylesheet is the preferred means of writing CSS, inline styling will override it if their CSS properties conflict.